mapeditor/inspector/inspector.cpp

```diff
@@ -905,6 +907,17 @@ QTableWidgetItem * Inspector::addProperty(CGCreature::Character value)
 	return item;
 }
 
+QTableWidgetItem * Inspector::addProperty(const std::optional<CGDwellingRandomizationInfo> & value)
+{
+	QString text = QObject::tr("Select town");
+	if(value.has_value() && !value->instanceId.empty())
```

Comment:
is it even possible that string exists but it's empty?

Reply:
Yes, editor initialize it with empty string, so it could exist like that on some maps